frontend/src/metabase/models/components/ModelDetailPage/ModelUsageDetails/ModelUsageDetails.tsx

```diff
@@ -66,4 +72,14 @@ function ModelUsageDetails({ model, cards }: Props) {
   );
 }
 
-export default Questions.loadList({ listName: "cards" })(ModelUsageDetails);
+function getCardListingQuery(state: State, { model }: OwnProps) {
+  return {
+    f: "database",
```

Comment:
what the `f` ? does it stand for `filter`?

Reply:
Good question I have no answer for unfortunately 😂 Found it in the endpoint docstring